AEP/aep-00001.md

```diff
@@ -0,0 +1,191 @@
+In AEPs, the "ArmoniK Developer Team" refers to the team at [Aneo](https://www.aneo.eu/) in charge of deciding the core functionnalities and direction of ArmoniK. This term does not extend to ArmoniK external contributors. The ArmoniK Developer Team decides whether a proposal should be considered as an AEP and they usually develop the feature described in them. More information about the ArmoniK Developer Team and ArmoniK Governance can be found in [AEP-00004](https://github.com/aneoconsulting/ArmoniK.Community/blob/main/AEP/aep-00004.md).
+
+## Submit an idea for ArmoniK
+
+The AEP process begins with a new idea for ArmoniK. It is highly recommended that a single AEP contain a single key proposal or new idea; the more focused the AEP, the more successful it tends to be. Most enhancements and bug fixes don’t need an AEP and can be submitted directly to the [ArmoniK issue tracker](https://github.com/aneoconsulting/ArmoniK/issues). The ArmoniK Developer Team reserves the right to reject AEP proposals if they appear too unfocused or too broad.
+
+To submit an idea for an AEP, open an issue on the [ArmoniK.Community issue tracker](https://github.com/aneoconsulting/ArmoniK.Community/issues) with the idea clearly presented. The discussion about the potential AEP is held in the issue. After reaching a consensus with the community and the ArmoniK Developer Team, the idea can move to the next stage. This step can be skipped by the ArmoniK Developer Team due to internal discussions.
+
+## AEP State Diagram
+
+An AEP can be in one of several states as described in this diagram :
+
+```mermaid
+flowchart LR
+    Draft-->Accepted
+    Accepted-->Final
+    Accepted-.->Rejected
+    Draft-.->Rejected
+    Final-.->Replaced
+    Draft-->Active
+    Active-.->Replaced
+    Active-.->Active
+```
+
+Transitions are described in the workflow below.
+
+## Creating the AEP
+
+Once a consensus is reached within the initial issue, the author forks the ArmoniK.Community and creates an `aep-XXXXX.md` file in the AEP directory, where XXXXX corresponds to the next available AEP number. The next available AEP number is a number which is not used in the main repository or is not used in an open pull request in the [ArmoniK.Community repository](https://github.com/aneoconsulting/ArmoniK.Community). The author then writes the AEP following the guidelines and rules in AEP-00001 (the present document). The discussions in the issue must be taken into account and should be referenced in the AEP. A pull request is then created to merge the fork into the [ArmoniK.Community repository](https://github.com/aneoconsulting/ArmoniK.Community).
+
+The discussions regarding the AEP are continued in the pull request itself. Suggested revisions to the AEP should include the potential adjustments to the language used (grammar, sentence structure...), specifications clarifications or any relevant point missing or unclear that need to be precised. During this time, the AEP is in a Draft state.
```

Comment:
We do not create a langage. We should adapt this sentence to ArmoniK

Reply:
I guess this sentence is not clear enough and should be rephrased. I'm talking about the language of the AEP itself (grammar, sentencing, tone...)